core/committer/txvalidator/v14/validator_test.go

```diff
@@ -1450,9 +1450,9 @@ func (m *mockLedger) GetBlockByTxID(txID string) (*common.Block, error) {
 }
 
 // GetTxValidationCodeByTxID returns validation code of give tx
-func (m *mockLedger) GetTxValidationCodeByTxID(txID string) (peer.TxValidationCode, error) {
+func (m *mockLedger) GetTxValidationCodeByTxID(txID string) (peer.TxValidationCode, uint64, error) {
```

Comment:
This is a change to a public method, but given Fabric is not designed to be a library, I don't think we need to document the API change.